manifest/v1alpha/report/report.go

```diff
@@ -0,0 +1,87 @@
+package report
+
+import (
+	"github.com/nobl9/nobl9-go/manifest"
+)
+
+//go:generate go run ../../../internal/cmd/objectimpl Report
+
+func New(metadata Metadata, spec Spec) Report {
+	return Report{
+		APIVersion: manifest.VersionV1alpha,
+		Kind:       manifest.KindReport,
+		Metadata:   metadata,
+		Spec:       spec,
+	}
+}
+
+type Report struct {
+	APIVersion manifest.Version `json:"apiVersion"`
+	Kind       manifest.Kind    `json:"kind"`
+	Metadata   Metadata         `json:"metadata"`
+	Spec       Spec             `json:"spec"`
+
+	Organization   string `json:"organization,omitempty"`
+	ManifestSource string `json:"manifestSrc,omitempty"`
+}
+
+type Metadata struct {
+	Name        string `json:"name" validate:"required,objectName"`
+	DisplayName string `json:"displayName,omitempty"`
+}
+
+type Spec struct {
+	CreatedAt          string                    `json:"createdAt,omitempty"`
+	UpdatedAt          string                    `json:"updatedAt,omitempty"`
+	Shared             bool                      `json:"shared" validate:"required"`
+	ExternalUserID     *string                   `json:"user,omitempty"`
```

Comment:
```suggestion
	CreatedBy     *string                   `json:"createdBy,omitempty"`
```
Could you please rename it so it would be aligned with other specs? See for example https://github.com/nobl9/nobl9-go/blob/main/manifest/v1alpha/slo/slo.go#L56